ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -648,12 +648,12 @@ private void printConnectorDescription(final ConnectorDescription description) {
     writer().println(String.format("%-20s : %s", "Type", description.getStatus().type()));
     writer().println(String.format("%-20s : %s", "State", status.connector().state()));
     writer().println(String.format("%-20s : %s", "WorkerId", status.connector().workerId()));
-    if (!status.connector().trace().isEmpty()) {
+    if (status.connector().trace() != null && !status.connector().trace().isEmpty()) {
```

Comment:
Yuck :(

Can we raise an upstream change to ensure this is never null, but an empty string?

```suggestion
    if (!ObjectUtils.defaultIfNull(status.connector().trace(), "").isEmpty()) {
```

Reply:
yeah I was surprised about that as well, not sure how I didn't catch it till now! if I have time I'll look into changing it upstream, but public API changes in AK...